qemu/tests/cfg/numa_prealloc_threads.cfg

```diff
@@ -0,0 +1,20 @@
+- numa_prealloc_threads:
+    no RHEL.6 RHEL.7 RHEL.8
+    no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8
+    required_qemu = [7.2,)
+    type = numa_prealloc_threads
+    virt_test_type = qemu
+    login_timeout = 1000
+    qemu_command_prefix = "taskset -c 0 "
+    vm_thread_contexts = "tc1"
+    smp_fixed = 8
+    vcpu_maxcpus = ${smp_fixed}
+    vm_thread_context_cpu-affinity = "1-7"
```

Comment:
To be more accurate, it is `vm_thread_context_cpu-affinity_tc1`

Reply:
@yanan-fu based on the current avocado-vt implementation, for `vm_thread_context_cpu-affinity_tc1` I would get a new `cpu-affinity_tc1=1-7` option in the QEMU cmd line, e.g.
`[stdlog]     -object thread-context,cpu-affinity_tc1=1-7,cpu-affinity=1-7,id=thread_context-tc1  \`

Reply:
Based on the following line, it is strange to get this result.
https://github.com/avocado-framework/avocado-vt/blob/01c4cb520de5263b70d9331ef904849e0c8d6fcc/virttest/qemu_vm.py#L1274C77-L1274C77

I will check it soon, thanks!

Reply:
Code updated following your review to make it clear for you
Ps: the case doesn't pass so don't merge this yet :grinning:

Reply:
The reason is there are 2 params after `object_params`: 
`vm_thread_context_cpu-affinity`  and `vm_thread_context_cpu-affinity_tc1`

So `tc_params` include the key `cpu-affinity` and `cpu-affinity_tc1`.
I am confused that if the `cpu-affinity` is a vm level configuration or just for a specified thread_context.
This is another topic for the current design in the avocado-vt, i am ok to leave it.

Then, for this patch,   yes, we do not use the suffic `tc1`, let's remove it.

Reply:
Updated again, thanks @yanan-fu for the review